README.md

```diff
@@ -1 +1,60 @@
-# Hello_Microverse
\ No newline at end of file
+Hello_Microverse
+
+_______________________________________________________________________________
+
+
+Additional description about the project and its features.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+
+_______________________________________________________________________________
+Live Demo
+Live Demo Link
+_______________________________________________________________________________
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+
+Setup
+
+Install
+
+Usage
+
+Deployment
```

Comment:
- [ ] As pointed out by the previous code reviewer, you need to personalize these details in the template not by just listing them. For example
**Setup**: Clearly explain how another developer can set up this project in his/her local machine.
**Built With**: Explain the major languages you have used to build this project.
**etc.**